index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Concert Festival Website">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>C Festival</title>
+  <link href="style.css" rel="stylesheet">
+  <link href="about.css" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css">
+</head>
+<body>
+  <div>
+    <nav>
+      <header class="container-fluid header pt-0 pb-0">
+        <ul class="list-unstyled list-inline text-end header-bar lead fs-6">
+          <li class="list-inline-item"><a href="#" aria-label="Facebook"><i class="header-item bi-facebook"></i></a></li>
+          <li class="list-inline-item"><a href="#" aria-label="Twitter"><i class="header-item bi-twitter"></i></a></li>
+          <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">English</a></li>
+          <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">My Page</a></li>
+          <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">Logout</a></li>
+        </ul>
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. `the typos, spacing(margin, padding)` of the elements on your page have to match the ones provided on the template
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203995035-5f3b026e-2743-4d82-ad46-bc9ae4544574.png)

Template provided :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203995148-372fa653-1380-441a-8c1c-d031eb7215c6.png)